fastlane/lib/fastlane/actions/device_grid/device_grid.rb

```diff
@@ -0,0 +1,102 @@
+# rubocop:disable Style/IndentationConsistency
+# We disable this rule since we intend opening HTML labels
+module Danger
+  class Dangerfile
+    module DSL
+      # A danger plugin: https://github.com/danger/danger
+      class DeviceGrid < Plugin
+        # @param languages: Array of languages you want to see (e.g. [en, de])
+        # @param devices: Array of deviecs you want to see (e.g. ["iphone4s", "ipadair"])
+        # @param prefix_command: Prefix the `fastlane run appetize_url_generator` command with something
+        #   this can be used to use `bundle exec`
+        def run(languages: nil, devices: nil, prefix_command: nil)
+          public_key = nil
+          [".fastlane/appetize_public_key.txt", "fastlane/appetize_public_key.txt"].each do |current| # not using Dir[...] since it doesn't show hidden files
+            public_key = File.read(current).strip if File.exist?(current)
+          end
+          UI.user_error!("No #{public_key_path} file found, make sure to run fastlane with `generate_device_grid` before calling `device_grid` in danger") if public_key.to_s.length == 0
+          File.delete(public_key_path)
+
+          devices ||= %w(iphone4s iphone5s iphone6s iphone6splus ipadair)
+          languages ||= ["en"]
+          prefix_command ||= ""
+
+          deep_link_matches = pr_body.match(/:link:\s(.*)/) # :link: emoji
+          deep_link = deep_link_matches[1] if deep_link_matches
+
+          ENV["FASTLANE_DISABLE_COLORS"] = "true" # since we fetch the URL from the output
+
+          markdown("<table>")
+          languages.each do |current_language|
+            markdown("<tr>")
+              markdown("<td>")
+                markdown("<b>#{current_language[0..1]}</b>")
+              markdown("</td>")
+
+              devices.each do |current_device|
+                markdown("<td>")
+
+                params = {
+                  public_key: public_key,
+                  language: current_language,
+                  device: current_device
+                }
+                params[:launch_url] = deep_link if deep_link
+                params_str = params.collect { |k, v| "#{k}:\"#{v}\"" }.join(" ")
+                url = `#{prefix_command} fastlane run appetize_url_generator #{params_str}`
+                url = url.match(%r{Result:.*(https\:\/\/.*)})[1].strip
+
+                markdown("<a href='#{url}'>")
+                  markdown("<img src='#{url_for_device(current_device)}' />")
+                  markdown("<p align='center'>#{beautiful_device_name(current_device)}</p>")
+                markdown("</a>")
+
+                markdown("</td>")
+              end
+            markdown("</tr>")
+          end
+          markdown("</table>")
+        end
+
+        def beautiful_device_name(str)
+          str = str.to_sym
```

Comment:
str.to_sym inline with the hash access and make this just a one-line function with no return